src/components/house/HouseJob.js

```diff
@@ -0,0 +1,216 @@
+            borderRightWidth: 1,
+            height: 100,
+            width: 120,
+          }}
+          >
+            <View style={{ flexDirection: 'row' }}>
+              <Text style={{
+                color: Colors.gray,
+                fontSize: 16,
+                fontWeight: '500',
+                marginRight: 6,
+              }}
+              >
+                {`#${index}`}
+              </Text>
+              <Image
+                style={{ width: 80, height: 25 }}
+                source={require('../../../assets/images/onlinerby.png')}
+              />
+            </View>
+            <View style={{
+              marginTop: 12,
+              flexDirection: 'row',
+            }}
+            >
+              <IonIcon
+                name={Platform.OS === 'ios' ? 'ios-search' : 'md-search'}
+                color={Colors.gray}
+              />
+              <Text style={{ color: Colors.gray, fontSize: 16, fontWeight: '500' }}>
+                { item.apartments && item.apartments.onliner && item.apartments.onliner.apartments ? item.apartments.onliner.apartments.length : '-0'}
+              </Text>
+            </View>
+          </View>
+          <View style={{ flexDirection: 'column', height: 80 }}>
+            <Text style={{
+              fontSize: 16,
+              fontWeight: '500',
+              color: Colors.gray,
+              marginLeft: 9,
+            }}
+            >
+            PRICE
+            </Text>
+            <View style={{
+              width: 110,
+              height: 35,
+              marginLeft: 9,
+              alignItems: 'center',
+              flexDirection: 'row',
+            }}
+            >
+              <IonIcon
+                name={Platform.OS === 'ios' ? 'ios-cash' : 'md-cash'}
+                color={Colors.gray}
+              />
+              <Text style={{ color: Colors.gray, fontSize: 16, fontWeight: '500' }}>
+                { item.params.min }
+              -
+                { item.params.max }
+              </Text>
+            </View>
+          </View>
+          <View style={{ flexDirection: 'column', height: 80 }}>
+            <Text style={{
+              fontSize: 16,
+              fontWeight: '500',
+              color: Colors.gray,
+              marginLeft: 9,
+            }}
+            >
+            ROOMS
+            </Text>
+            <View style={{
+              width: 70,
+              height: 35,
+              marginLeft: 9,
+              flexDirection: 'row',
+              alignItems: 'center',
+            }}
+            >
+              <IonIcon
+                name={Platform.OS === 'ios' ? 'ios-home' : 'md-home'}
+                color={Colors.gray}
+              />
+              <Text style={{ color: Colors.gray, fontSize: 16, fontWeight: '500' }}>
+                { item.params.rooms.toString() }
+              </Text>
+            </View>
+            <TouchableOpacity
+              onPress={() => {
```

Comment:
Don't use the anonymous functions
Create a class method

Reply:
The absence of anonymous function as a wrapper triggers one false triggering of an event.